src/nxdoc/nuxeo-server/administration/elasticsearch-setup.md

```diff
@@ -516,31 +512,14 @@ In `/etc/default/elasticsearch` file you can increase the JVM heap to half of th
 ES_JAVA_OPTS="-Xms6g -Xmx6g"
 ```
 
-To prevent indexing errors like:
-
-```
-EsRejectedExceptionException[rejected execution (queue capacity 50)
-```
-
-Increase the bulk queue size In`/etc/elasticsearch/elasticsearch.yml` configuration file:
-
-```
-# For Elasticsearch 2.x
-# threadpool.bulk.queue_size: 500
-# For Elasticsearch 5.6
-# thread_pool.bulk.queue_size: 500
-# For Elasticsearch 6.x
-thread_pool.write.queue_size: 500
-```
-
-To reduce disk IO you should consider changing the default [translog](https://www.elastic.co/guide/en/elasticsearch/reference/6.3/index-modules-translog.html)
+To reduce disk IO you should consider changing the default [translog](https://www.elastic.co/guide/en/elasticsearch/reference/8.7/index-modules-translog.html)
```

Comment:
we should point to 7.x docs which is the main target, we do support the 8.x but only with 7.x compatibility mode